docs/kluctl/templating/variable-sources.md

```diff
@@ -273,6 +273,21 @@ vars:
 The advantage of the latter is that the auto-generated suffix in the ARN (which might not be known at the time of
 writing the configuration) doesn't have to be specified.
 
+### gcpSecretsManager
```

Comment:
Can you rename all occrencies of gcpSecretsManager to gcpSecretManager (including the struct name)? I know that it's plural for AWS, but that is simply because all docs in the AWS world use the plural form. For GCP, it seems to be always the singular form in the docs.